openlcsd/flow/tasks.py

```diff
@@ -944,11 +945,12 @@ def get_module_components_from_corgi(context, engine):
     """
     Get module components from corgi
     """
-    config = context.get('config')
+    config = context.get("config")
     engine.logger.info("Start to get module components data...")
-    mc = ParentComponentsAsync(
-        config.get('CORGI_API_PROD'), context.get('package_nvr'))
-    context['components'] = mc.get_components_data('RHEL_MODULE')
+    mc = CorgiConnector(
+        config.get("CORGI_API_PROD"))
+    nvr = context.get("package_nvr")
+    context["components"] = mc.get_components_data(nvr, "RHEL_MODULE")
```

Comment:
Same as the last comment, the 'RHEL_MODULE' has been updated to 'RPMMOD' on corgi production.